contracts/core/OptionsPremiumPricerInStables.sol

```diff
@@ -69,31 +69,80 @@ contract OptionsPremiumPricer is DSMath {
         uint256 expiryTimestamp,
         bool isPut
     ) external view returns (uint256 premium) {
+        uint256 spotPrice = priceOracle.latestAnswer();
+        (uint256 assetPrice, uint256 assetDecimals) =
+            isPut
+                ? (stablesOracle.latestAnswer(), stablesOracleDecimals)
+                : (spotPrice, priceOracleDecimals);
+
+        premium = _getPremium(
+            st,
+            expiryTimestamp,
+            spotPrice,
+            assetPrice,
+            assetDecimals,
+            isPut
+        );
+    }
+
+    /**
+     * @notice Calculates the premium of the provided option using Black-Scholes in stables
+     * @param st is the strike price of the option
+     * @param expiryTimestamp is the unix timestamp of expiry
+     * @param isPut is whether the option is a put option
+     * @return premium for 100 contracts with 18 decimals
+     */
+    function getPremiumInStables(
+        uint256 st,
+        uint256 expiryTimestamp,
+        bool isPut
+    ) external view returns (uint256 premium) {
+        premium = _getPremium(
+            st,
+            expiryTimestamp,
+            priceOracle.latestAnswer(),
+            stablesOracle.latestAnswer(),
+            stablesOracleDecimals,
+            isPut
+        );
+    }
+
+    /**
+     * @notice Internal function to calculate the premium of the provided option using Black-Scholes
+     * @param st is the strike price of the option
+     * @param expiryTimestamp is the unix timestamp of expiry
+     * @param spotPrice is the spot price of the underlying asset
+     * @param assetPrice is the denomination asset for the options
+     * @param assetDecimals is the decimals points of the denomination asset price
+     * @param isPut is whether the option is a put option
+     * @return premium for 100 contracts with 18 decimals
+     */
+    function _getPremium(
+        uint256 st,
+        uint256 expiryTimestamp,
+        uint256 spotPrice,
+        uint256 assetPrice,
+        uint256 assetDecimals,
+        bool isPut
+    ) internal view returns (uint256 premium) {
```

Comment:
Let's maintain consistency in the order of params for the two `_getPremium` https://github.com/ribbon-finance/rvol/pull/32/files#diff-5db42908784c806211ef293c5ae8454b3bf186288dc12ea7bef3ea6a10adfb4aR128